src/transformers/models/prophetnet/modeling_prophetnet.py

```diff
@@ -859,11 +859,7 @@ def forward(
     ):
         batch_size, ngram_sequence_length, hidden_size = hidden_states.size()
 
-        assert list(hidden_states.size()) == [
-            batch_size,
-            ngram_sequence_length,
-            hidden_size,
-        ], (
+        assert list(hidden_states.size()) == [batch_size, ngram_sequence_length, hidden_size], (
             f"`hidden_states` should be of shape {batch_size, ngram_sequence_length, hidden_size}, but is of shape"
             f" {hidden_states.shape}"
         )
```

Comment:
Seems unrelated, and if we are going to change it, let's make it a real test and an exception raised :-p

Reply:
I'm pretty sure it's my black that does this. However it should be 22.3 Like `transformers expect. Not sure where the diff comes from.

And `fixup` doesn't revert those :(